MaterialSkin/HTML/EN/html/material-skin/index.html.zh_tw

```diff
@@ -0,0 +1,104 @@
+[% PROCESS materialhelpheader.html %]
+<h1>Introduction</h1>
+<p>MaterialSkin 是一款專為 LMS 開發的 HTML5 應用程式，主要提供以下兩種佈局：
+  <ul>
+    <li>移動裝置佈局</li>
+    <li>桌上型裝置佈局</li>
+  </ul>
+在預設情況下，此程式會為您的螢幕寬度選擇最佳的佈局，但如果您需要更改佈局時可以前往 '設定 -> 界面' 來對目前或其他播放器或進行強制修改。
+</p>
+<p>這個幫助頁面會提供ㄧ些基本資訊，幫助您了解
+此應用程式的一些詳細用法。
+</p>
+<br/>
+
+<h1>移動裝置佈局</h1>
+<p>此佈局是專為移動裝置如手機或是小平版所設計的，此佈局將在底部放置視窗的切換欄來進行其他視窗的切換，可切換的視窗如下：
+  <ul>
+    <li>瀏覽視窗 - 瀏覽媒體庫中的收藏、應用程式或電台等等。</li>
+    <li>播放視窗 - 顯示正在播放中的曲目資訊如標題、封面圖片等等。</li>
+    <li>佇列視窗 - 顯示佇列中的曲目清單。</li>
+  </ul>
+您可以透過輕觸底部視窗切換或是在螢幕中間左右滑動來切換視窗。如果直接輕觸底部切換欄中當前視窗的按鈕將會有以下功能：
+  <ul>
+    <li瀏覽視窗中 – 會返回上一層，與輕觸返回鍵有一樣的效果。</li>
+    <li>播放視窗中 – 如果 LMS 有安裝插件 'Music and Artist Information' 則會顯示演出者、專輯和曲目資訊。</li>
+    <li>佇列視窗中 – 會滾動至目前播放的曲目。</li>
+  </ul>
+</p>
+<br/>
+
+<h1>桌上型裝置佈局</h1>
+<p>此佈局是專為桌上型裝置如個人電腦或是筆記型電腦所設計的，將畫面在左側放置瀏覽視窗，右側放置佇列視窗，如果有正在播放的歌曲則會將相關資訊顯示
+在底部。透過輕觸主工具列的’展開目前播放’按鈕會將詳細的曲目資訊和大型的封面圖片顯示在主視窗中，同樣地，如果安裝了
+'Music and Artist Information' 插件，輕觸’顯示目前播放曲目’按鈕則會在畫面顯示包含演出者、專輯和歌詞等的詳細資訊。
```

Comment:
Should this not also be translated?

Reply:
It's plugin related, should be translate later after the slimserver is translated.